srcpkgs/i3lock-color/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'i3lock-color'
+pkgname=i3lock-color
+version=2.11
+revision=1
+wrksrc="${pkgname}-${version}-c"
+only_for_archs="i686 x86_64"
```

Comment:
why is this such ? it certainly doesn't seem like it

Reply:
I've only built and tested the package on x86_64. I noticed the AUR package for this had it listed as only being for the archs I listed above which is why I had it. 

I've removed the line.

Reply:
Arch Linux just builds for x86_64, and on occasion where 32bit packages for x86_64 environments benefit (like wine) i686.

Void Linux builds for x86_64, i686, armv[56], aarch64.

only_for_arches should be used in occasions where like upstream doesn't support that architecture and it is broken.